src/ProjectFileTools/extension.vsixmanifest

```diff
@@ -13,6 +13,7 @@
   <Assets>
     <Asset Type="Microsoft.VisualStudio.VsPackage" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
     <Asset Type="Microsoft.VisualStudio.MefComponent" Path="|ProjectFileTools.NuGetSearch|" />
+    <Asset Type="Microsoft.VisualStudio.MefComponent" Path="|ProjectFileTools.MSBuild|" />
```

Comment:
In Adrian's comment he was confirming that we did not need to add the dll as an asset. The current structure compiles and runs correctly.